src/analytics/index.tsx

```diff
@@ -0,0 +1,41 @@
+import {
+  sendAnalyticsEvent as sendAnalyticsTraceEvent,
+  Trace as AnalyticsTrace,
+  TraceEvent as AnalyticsEvent,
+} from '@uniswap/analytics'
+import { atomWithStorage, useAtomValue } from 'jotai/utils'
+import { memo } from 'react'
+
+const analyticsOptOutAtomName = 'optOutOfAnalytics'
+export const analyticsOptOutToggle = atomWithStorage<boolean>(analyticsOptOutAtomName, true)
```

Comment:
This reads like its negation. If you keep this name, you should invert the default value and all usages. Otherwise, something like 'allow_analytics'.

```suggestion
const analyticsOptOutKey = 'analytics_opt_out'
export const analyticsOptOutAtom = atomWithStorage<boolean>(analyticsOptOutKey, false)
```